app/src/ui/dialog/dialog.tsx

```diff
@@ -467,7 +470,17 @@ export class Dialog extends React.Component<DialogProps, IDialogState> {
     // anchor tag masquerading as a button)
     let firstTabbable: HTMLElement | null = null
 
-    const closeButton = dialog.querySelector(':scope > header button.close')
+    const closeButton = dialog.querySelector(
+      ':scope > div.dialog-header button.close'
```

Comment:
Ouch! Nice catch!

Reply:
Which this means if there was a dialog where there were no first suitable focus elements.. it was no longer working as expected and focusing the x... but I believe all of our dialogs have a cancel and/or submit button, thus, why we didn't realize earlier that that querySelector didn't work.